### ISSUE TITLE
Flatpak

### ISSUE THREAD

**Issue by PopeRigby** [radicle-dev/radicle-upstream#2093]
It would be cool if Radicle Upstream was published as a Flatpak on Flathub, because it makes it easy to distribute to all Linux distributions. In contrast to AppImages, a Flatpak is easier to keep up-to-date (you manage them via the `flatpak` CLI tool, or a frontend), and to integrate with the system (`.desktop` file, etc.)

**Comment by geigerzaehler** [comment]
I agree, Flatpak would be preferable over AppImage. The next steps for this would be
1. Figure out how easy it is to build Flatpak with electron-builder
2. Figure out how distributing Flatpak on Flathub (or other platforms) works. We need to look in detail at ensuring integrity and authenticity.
3. Determine the necessary changes to our release process and current distribution mechanism
4. Provide a rough estimate to implement these changes.
